.eslintrc.yml

```diff
@@ -40,5 +40,18 @@ rules:
     - error
     - devDependencies: true
   padded-blocks: 0
-  key-spacing:
-    - 0
+  key-spacing: 0
+  space-before-function-paren:
+    - 2
+    - never
+  func-names:
+    - 2
+    - as-needed
+  react/prefer-stateless-function:
+    - 2
+    - ignorePureComponents: true
+  react/forbid-prop-types:
+    - 2
+    - forbid:
+      - any
+      - array
```

Comment:
OK, I was wondering why array was bad. It seems like it might be something we want to allow. I'm fine forbidding 'any'